components/automate-ui/src/app/entities/cookbooks/cookbook-details.reducer.ts

```diff
@@ -0,0 +1,46 @@
+import { EntityState, EntityAdapter, createEntityAdapter } from '@ngrx/entity';
+import { set } from 'lodash/fp';
+
+import { EntityStatus } from 'app/entities/entities';
+import { CookbookDetailsActionTypes, CookbookDetailsActions } from './cookbook-details.actions';
+import { CookbookDetails } from './cookbook-details.model';
+
+export interface CookbookDetailsEntityState extends EntityState<CookbookDetails> {
+  status: EntityStatus;
```

Comment:
This `status` is unused in this file. Is it used somewhere else? If so, we should change to a qualified name (like deleteStatus, getAllStatus, getStatus, etc.)